### ISSUE TITLE
What is TELinkTypeComplex for?

### ISSUE THREAD

**Issue by nariakiiwatani** [TouchDesigner/TouchEngine-Windows#2]
Hi, thank you for the great work!
I'm now developing [ofxTouchEngine](https://github.com/nariakiiwatani/ofxTouchEngine) which is a TouchEngine wrapper for openFrameworks.

`Multiple links which collectively form a single complex link` in `TEInstance.h` seems to be a explanation for `TELinkTypeComplex`.
Yes I know what it says but I couldn't figure out which feature in TouchDesigner corresponds to it.
In other words, how I can create `TELinkTypeComplex` in my `.tox`?

Any suggestion would help!

**Comment by bangnoise** [comment]
Great to see an ofxAddon in the works!

TELinkTypeComplex you can safely ignore for now - we haven't defined any instances of it yet. When we do, their use will be documented.

**Comment by nariakiiwatani** [comment]
@bangnoise thank you for the answer!
If my guess is correct, it would be exactly what I want in TD.
I'm looking forward to it!
